File: WorldFeed/Server/src/WorldFeed.Common/Models/Properties/Trend.cs
namespace WorldFeed.Common.Models.Properties
{
    using Newtonsoft.Json;

    using global::WorldFeed.Common.Public.Models.Interfaces;

    public class Trend : ITrend
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("url")]
        public string URL { get; set; }

        [JsonProperty("query")]
        public string Query { get; set; }

        [JsonProperty("promoted_content")]
        public string PromotedContent { get; set; }

        [JsonProperty("tweet_volume")]
        public long? TweetVolume { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Core/Models/Properties/Trend.ts
import {ITrend} from "../../../Public/Models/Interfaces/ITrend";

export class Trend implements ITrend {
  // [JsonProperty("name")]
  public name: string;

  // [JsonProperty("url")]
  public URL: string;

  // [JsonProperty("query")]
  public query: string;

  // [JsonProperty("promoted_content")]
  public promotedContent: string;

  // [JsonProperty("tweet_volume")]
  public tweetVolume: number; // long?
}
